ocp_resources/grafana.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class Grafana(NamespacedResource):
```

Comment:
Empty class without to_dict implement required is not allowed.
Please introduce to_dict().